### PR TITLE
PX-Backup 1.0.1

### PR DESCRIPTION
**Pull request by nathan-px**
* Release notes
* Remove CSI

### CODE REVIEW

File: content/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,53 @@ disableprevnext: true
...
+| PB-499 | When PX-Backup failed to create a backup, the UI became stuck, showing only the creation failure message. <br/><br/>**User Impact:**  Due to the stuck message, users may have been unable to see the real reason for a backup deletion failure.<br/><br/>**Resolution:** Creation failure messages no longer stick, allowing users to see more recent messages. |
+| PB-493 |The PX-Backup UI did not update to reflect invalid underlying backup object statuses.<br/><br/>**User Impact:** If a backup object was deleted, users may still have seen the backup appear as valid on the PX-Backup UI.<br/><br/>**Resolution:** PX-Backup now properly displays the invalid backup status. |
+| PB-468 | Users could create more backups than their license allows.<br/><br/>**User Impact:** Users who created more backups than their license allows would experience backup failures and error messages.<br/><br/>**User Impact:** PX-Backup now stops users from creating excess backups before they exceed their license capacity. |
+
+
+<!-- I don't think these need release notes
```
Comment (nathan-px) [inline content/release-notes.md:40]: @bhavana-ar I think all of these were flagged as "release notes not required", I tried my best to pick ones that looked like they may have needed a RN, but we should review.

File: content/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,53 @@ disableprevnext: true
...
+June 1, 2020
+
+### Improvements
+
+Portworx, Inc. has upgraded or enhanced functionality in the following areas:
+
+| **Improvement Number** | **Improvement Description** |
+|----|----|
+| PB-547 | PX-Backup now allows more than 12 backups to be retained while creating a schedule. |
+| PB-515 | Users can now add and edit a cluster to Lighthouse independently from PX-Backup. <!-- need to confirm this. --> |
```
Comment (nathan-px) [inline content/release-notes.md:22]: @bhavana-ar I don't really understand this, let's chat about it.

File: content/use-px-backup/upgrade/_index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+Upgrade PX-Backup by entering the following command:
+
+```text
+curl 
+```
+
+This command retrieves the upgrade script and runs it. 
+<!-- Need the command! it's piped curl to pull and execute -->
```
Comment (nathan-px) [inline content/use-px-backup/upgrade/_index.md:19]: @bhavana-ar need the command here

File: content/use-px-backup/upgrade/upgrade-script-reference.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+description: Reference information for the PX-Backup upgrade script
+keywords: upgrade, reference,
+weight: 8
+hidesections: true
+disableprevnext: true
+scrollspy-container: false
+type: common-landing
+---
+
+<!-- I don't think this is specific to backup, is it? we should puth this in the PX-Central on-prem docs. -->
```
Comment (nathan-px) [inline content/use-px-backup/upgrade/upgrade-script-reference.md:12]: @bhavana-ar the upgrade script isn't specific to PX-Backup, is it?

File: content/release-notes.md [hunk 5]
```diff
@@ -8,6 +8,53 @@ disableprevnext: true
...
+| PB-307 | Restores from subsequent backups stored on different buckets from the original that both involve the same namespace failed. <!-- this is tough to explain and I'm not sure I got it right. -->  <br/><br/>**User Impact:** If users created multiple backups and buckets, restores from subsequent backups stored on different buckets from the original that both involve the same namespace failed.<br/><br/>**Resolution:** PX-Backup now properly takes the initial backup for incremental backups, even when they share a namespace with other backups using different buckets. |
+| PB-499 | When PX-Backup failed to create a backup, the UI became stuck, showing only the creation failure message. <br/><br/>**User Impact:**  Due to the stuck message, users may have been unable to see the real reason for a backup deletion failure.<br/><br/>**Resolution:** Creation failure messages no longer stick, allowing users to see more recent messages. |
```
Comment (bhavana-ar) [inline content/release-notes.md:46]: Reason for backup deletion failure when backup creation had failed was not getting propagated to the UI correctly

File: content/release-notes.md [hunk 6]
```diff
@@ -8,6 +8,53 @@ disableprevnext: true
...
+| PB-493 |The PX-Backup UI did not update to reflect invalid underlying backup object statuses.<br/><br/>**User Impact:** If a backup object was deleted, users may still have seen the backup appear as valid on the PX-Backup UI.<br/><br/>**Resolution:** PX-Backup now properly displays the invalid backup status. |
+| PB-468 | Users could create more backups than their license allows.<br/><br/>**User Impact:** Users who created more backups than their license allows would experience backup failures and error messages.<br/><br/>**User Impact:** PX-Backup now stops users from creating excess backups before they exceed their license capacity. |
```
Comment (bhavana-ar) [inline content/release-notes.md:48]: After the trial license expired, there was no way to let the user know that they had more clusters online on PX backup than the license supported until the backup failed.

Comment (bhavana-ar) [reply content/release-notes.md:48]: User Impact: User would not know they have more cluster than the license supports until their backup or restore failed.

Comment (bhavana-ar) [reply content/release-notes.md:48]: Resolution: A message is now displayed on the home screen to let the users know that they need to remove some clusters if they have more clusters than their license supports.

File: content/release-notes.md [hunk 7]
```diff
@@ -47,10 +50,9 @@ Portworx, Inc. has fixed the following issues:
 | PB-486 | When PX-Backup deleted a backup, the status would show as "DELETING", but the message would only show the previous message: "Volumes and resources were backed up successfully"<br/><br/>**User Impact:** Users may have seen confusing information about their backup status during deletion<br/><br/>**Resolution:** PX-Backup now shows an appropriate message indicating volumes and resources are being deleted. |
```
Comment (bhavana-ar) [inline content/release-notes.md:50]: Replace "When PX-Backup deleted a backup, the status would show as "DELETING", but the message would only show the previous message: "Volumes and resources were backed up successfully"  with "When user deleted a backup, the reason was not getting updated"

File: content/use-px-backup/upgrade/_index.md (hunk 3 above)
Comment (bhavana-ar) [reply content/use-px-backup/upgrade/_index.md]: @tusharraut1994 Can provide the curl command to fetch the upgrade script

File: content/use-px-backup/configure-users/_index.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+weight: 2
+hidesections: true
+disableprevnext: true
+scrollspy-container: false
+type: common-landing
+---
+
+Your non-admin PX-Backup users need certain Kubernetes permissions to perform backup and restore operations on your cluster and through PX-Central. Ensure your users have the following minimum cluster roles:
+
+*
```
Comment (nathan-px) [inline content/use-px-backup/configure-users/_index.md:14]: @bhavana-ar

**Pull request merged**